src/main/webapp/app/admin/standardized-competencies/knowledge-area-detail.component.html

```diff
@@ -8,32 +8,27 @@ <h4 jhiTranslate="artemisApp.knowledgeArea.details.titleCreate"></h4>
             }
             <button type="button" class="btn-close" (click)="close()"></button>
         </div>
-        <!-- do not allow the creation of children/competencies when creating a new knowledge area -->
+        <!-- do not allow these actions when creating a new knowledge area -->
```

Comment:
```suggestion
        <!-- do not allow these actions when creating a new knowledge area because we use the same UI area for it -->
```

Reply:
We don't use the same UI area though, we display them below. It just makes no sense for these actions to appear on a new knowledge area (not yet saved to db) because it is already getting edited, cannot be deleted in the db, and it also cannot have children or compentencies linked to it.

Reply:
I was thinking of like we create quiz exercises. You can also create multiple questions before saving. But I can see this getting out of hand with these nested structures.

Reply:
Maybe still add an explanation on "why"